source/max.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
 const max = numbers => Math.max(...numbers);
+
+const inverse = (array, ind) => {
+let numbers = array.slice();
+	if (ind) {
+		const len = numbers.length;
+		if (ind >= 0){
+			let reversedSubarray = numbers.slice(ind).reverse();
+			return numbers.slice(0, ind).concat(reversedSubarray);
+		} else {
+			let subarray = numbers.slice(ind);
+			return numbers.slice(0, len+ind).reverse().concat(subarray);
```

Comment:
тут можно сделать
```suggestion
			return numbers.slice(0, ind).reverse().concat(subarray);
```
и произойдет тоже самое - https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

и да, к какой мысли я веду - вне зависимости от того ind положительный или отрицательный создаешь два массива `numbers.slice(ind)` и `numbers.slice(0, ind)` и в зависимости от знака `ind` один из массивов переворачиваешь и склеиваешь в том же порядке - вот именно то что я написал буквами я и хочу увидеть в коде

Reply:
да, сейчас попробую

Reply:
Сделал